README.md

```diff
@@ -33,11 +39,31 @@ In order to flash and run these programs on an ESP32 device, you will need, in a
 * Some example applications require the presence of a WiFi Access Point (AP), to which the ESP32 device can connect.
 * Some example applications require either an existing WiFi Access Point (AP) or that your personal computer be capable of connecting to the ESP32 acting as a WiFi Access Point (AP).
 
-> Note. AtomVM is currently qualified only on the [ESP32](https://www.espressif.com/en/products/socs/esp32) SoC.  Support for the ESP32-C and ESP32-S* SoCs is TBD.
+> For information about supported ESP32 SoCs, see the AtomVM [Release Notes](https://www.atomvm.net/doc/master/release-notes.html).
 
 ### STM32 Requirements
 
-TODO
+In order to flash and run these programs on an STM32 device, you will need, in addition:
+
+* An STM32 device with support for USB connectivity via UART, such as the [STMicroelectronics](https://www.st.com) [STM32F4Discovery](https://www.st.com/en/evaluation-tools/stm32f4discovery.html).
+* Installation of the release of the AtomVM virtual machine image on the STM32 device.  For information about how to install a release of the AtomVM virtual machine image on an STM32 device, see the AtomVM [Getting Started Guide](https://doc.atomvm.net/getting-started-guide.html) documentation.
+* A USB cable to connect your STM32 device.
+* The [st-flash](https://github.com/texane/stlink) flashing tool.
```

Comment:
Also need hardware `st-link`. (as in release notes) This is included on both Nucleo and Discovery families of boards, but others require a external st-link device.